modules/ROOT/pages/_partials/afm-concepts.adoc

```diff
@@ -0,0 +1,73 @@
+// Reused in index.adoc and specific feature topics
+
+// tag::afm-monitors[]
+
+A monitor is a repeating schedule of one or more tests. Monitors might also be referred to as test suites. 
```

Comment:
Is there a way you can write all of these definitions without repeating the term in the first phrase? For example:

A monitor is a repeating schedule... > A repeating schedule...

The definition of test might present a challenge to doing this but maybe you can think of something.

Reply:
The reason I repeat the phrase is that I use some of these definitions in other contexts where it wouldn't make sense without it.